src/helpers/displayToast.tsx

```diff
@@ -0,0 +1,9 @@
+import { toast } from 'react-toastify';
+
+export const displayToastError = (title: string) => {
+  toast.error(title);
+};
+
+export const displayToastSuccess = (title: string) => {
```

Comment:
I wonder it's almost the same as `export const displayToastSuccess = toast.success;`, which means that we don't really need this additional function 🤔  and we can use toast.success inside needed logic.

But in the same time we have this import only once and it will be easy to manage library calls or even replace the library.

Reply:
Yes, I agree. Should I leave it as it is for now?

Reply:
Sure 👍